### ISSUE TITLE
Graphics export filter options for jpeg

### ISSUE THREAD

**Issue by keyliaran** [unoconv/unoconv#507]
I'm converting efm file to jpg (0.8.2 version of unoconv) using the following command:
./unoconv -f jpg --stdout  --export=Quality=100 --export=Resolution=600 page1.emf > out.jpg
The Quality options works well, but it's seems that Resolution not working at all.

**Comment by regebro** [comment]
You are correct, it makes no difference. It does if you specify it in the filter option dialog box, then the pixel density is saved correctly in the resulting JPEG metadata.

So what is the correct format for this option? Well, it's LibreOffice, so documentation is scarce, so I don't know. Sorry.

**Comment by regebro** [comment]
No response.
